src/app/core/auth-validation.service.ts

```diff
@@ -1,5 +1,5 @@
 import { Injectable } from '@angular/core';
-import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { AbstractControl, AsyncValidatorFn, FormControl, FormGroup, Validators } from '@angular/forms';
```

Comment:
```suggestion
import { FormControl, FormGroup, Validators } from '@angular/forms';
```
If these imports are not being used, lets delete them.